src/main/java/com/uditagarwal/algoritms/DoublyLinkedList.java

```diff
@@ -14,12 +15,18 @@
 @Getter
```

Comment:
We will have to remove this getter now. Otherwise it will expose `getDummyHead` and `getDummyTail`